cumulus/ansible/tasks/volume.py

```diff
@@ -38,16 +38,19 @@ def attach_volume(profile, cluster, instance, volume, path,
         'girder_cluster_id': cluster['_id'],
         'region': profile['regionName'],
         'profile_id': profile['_id'],
-        'volume_id': volume['ec2']['id'],
         'volume_name': volume['name'],
-        'volume_size': volume['size'],
         'volume_zone': volume['zone'],
         'instance_id': instance['instance_id'],
         'path': path,
         'ansible_ssh_private_key_file': _key_path(profile),
         'ansible_user': cluster['config']['ssh']['user']
     }
 
+    if volume['ec2']['id']:
```

Comment:
Is 'ec2.id. always part of a cluster? We might way the use jsonpath the extract out this value.